docs/discover/search-sessions.asciidoc

```diff
@@ -52,7 +52,7 @@ image::images/search-session-awhile.png[Search Session indicator displaying the
 Once you save a search session, you can start a new search,
 navigate to a different application, or close the browser.
 
-. To view your saved searches, go to the
+. To view your saved search sessions, go to the
 *Search Sessions* management page using the navigation menu or the <<kibana-navigation-search,global search field>>.
```

Comment:
Is this being renamed to the "Discover Sessions" management page in the UI as well?

Reply:
`search sessions` is another feature and it has *Search Sessions* management page. 

It's not related to this PR. Just fixed as I found it.

To answer your question: there is no "Discover Sessions" ("Saved Search") management page.

Reply:
Okay, thanks!